packages/xo-web/src/common/xo/index.js

```diff
@@ -821,6 +821,9 @@ export const getHostMissingPatches = async host => {
       ? patches
       : filter(patches, { paid: false })
   }
+  if (host.power_state !== 'Running') {
```

Comment:
Should remove this [resolveId](https://github.com/vatesfr/xen-orchestra/pull/5140/commits/f590e702cc01a739d9a3a864f2b6c941e1f8df4e#diff-c01aeb73effa527f784b11c39d4638e9R816)?

Reply:
I'd say no:
- it's not related to this PR
- it doesn't change the behaviour since we were already assuming that `host` was always an object in this method